src/pages/scheduler/components/Toolbar.tsx

```diff
@@ -14,11 +14,28 @@ export const CalendarToolBar =
         onSelectedDateChange(new Date());
       }
     };
+    console.log(vCalendar);
```

Comment:
In released version is this something that's needed to be logged

Reply:
oops